### ISSUE TITLE
Config ignores cache: redis: enabled: false

### ISSUE THREAD

**Issue by MysticRyuujin** [emeraldpay/dshackle#16]
```
cache:
  redis:
    enabled: false <---- FALSE
    host: localhost
    port: 6379
    db: 0
    password: "none"
```

```
2020-30-05 18:44:19.714 | INFO  |        CachesFactory | Use Redis cache at: redis://localhost
2020-30-05 18:44:20.512 | ERROR |        CachesFactory | Unable to establish connection to the Redis server
2020-30-05 18:44:20.512 | ERROR |        CachesFactory | Redis error: Unable to connect to localhost:6379
```

**Comment by splix** [comment]
fix 04958784ec

**Comment by MysticRyuujin** [comment]
I don't think this is fixed...

```
v0.7.2 built from 881c8c66d7 on 2020-07-03T02:53:12 UTC

2020-06-07 01:52:47.341 | INFO  |            StarterKt | Starting StarterKt on a9b0c2779aa8 with PID 1 (/app/classes started by root in /)
2020-06-07 01:52:47.345 | INFO  |            StarterKt | No active profile set, falling back to default profiles: default
2020-06-07 01:52:49.356 | INFO  |               Config | Using config: /etc/dshackle/dshackle.yaml
2020-06-07 01:52:49.489 | INFO  |        CachesFactory | Use Redis cache at: redis://localhost
2020-06-07 01:52:50.338 | ERROR |        CachesFactory | Unable to establish connection to the Redis server
2020-06-07 01:52:50.338 | ERROR |        CachesFactory | Redis error: Unable to connect to localhost:6379
2020-06-07 01:52:50.339 | ERROR |        CachesFactory | Redis config:
2020-06-07 01:52:50.339 | ERROR |        CachesFactory |   uri: localhost:6379
2020-06-07 01:52:50.339 | ERROR |        CachesFactory |   db: 0
2020-06-07 01:52:50.340 | ERROR |        CachesFactory |   password: (set)
2020-06-07 01:52:50.340 | ERROR |        CachesFactory | Stopping the server...
```

**Comment by splix** [comment]
fixed, but wasn't ported to 0.7.x release yet
